.circleci/continue_config.yml

```diff
@@ -11,6 +11,7 @@ jobs:
         default: false
     working_directory: ~/Empirical-Core
     parallelism: 8
+    resource_class: xlarge
```

Comment:
grpc compilation is longer than the CircleCI build timeout (which is not extensible unless we buy the `scale` CircleCI plan). This is an open issue (https://github.com/grpc/grpc/issues/35874)

Increasing the resource class solves the issue for now. One approach I could do is build grpc in a separate build step and cache it. Afterwards, reduce the `resource_class` to normal. That way, the built grpc will not be cache-invalidated by changes to Gemfile. Curious what peoples' thoughts are.